src/marqo/tensor_search/validation.py

```diff
@@ -138,18 +140,15 @@ def validate_field_content(field_content: Any, is_non_tensor_field: bool) -> Any
     Raises:
         InvalidArgError if field_content is not acceptable
     """
-    if type(field_content) in constants.ALLOWED_CUSTOMER_FIELD_TYPES:
+    if type(field_content) in constants.ALLOWED_UNSTRUCTURED_FIELD_TYPES:
```

Comment:
Do this validation in `UnstructuredVespaIndex` class. Note this method is also used by structured index add docs, which highlights the perils of these utility-type functions (coupling that is really easy to miss -> regression)